commands.go

```diff
@@ -1094,8 +1094,14 @@ type ZRangeByScore struct {
 	Offset, Count int64
 }
 
-func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores bool) *StringSliceCmd {
-	args := []interface{}{"ZRANGEBYSCORE", key, opt.Min, opt.Max}
+func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores, isLex bool) *StringSliceCmd {
```

Comment:
Instead of passing `isLex`, why not just rename the function to `zRangeByGeneric` and pass the actual `zcmd` string?